composer.json

```diff
@@ -20,21 +20,17 @@
         "symfony/console": "^4.0",
         "symfony/dependency-injection": "^4.0",
         "symfony/finder": "^4.0",
-        "symplify/easy-coding-standard": "^3.2",
-        "symplify/package-builder": "^3.2",
-        "webmozart/assert": "^1.3"
+        "symplify/package-builder": "^3.2"
     },
     "require-dev": {
+        "symplify/easy-coding-standard": "^3.2",
```

Comment:
@carusogabriel This might make you happy :) 
After bit of thinking this would be too much to add to phar at the moment, since ECS depends on 2 other big packages